release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -957,13 +973,35 @@ module openconfig-if-ip {
     }
 
     leaf suppress {
+      status deprecated;
       type boolean;
       default false;
       description
         "When set to true, router advertisement neighbor discovery
         messages are not transmitted on this interface.";
```

Comment:
This leaf is deprecated, but I thik it is stil valid to explain expectation.
Pls. consider adding to description" ..on this interface, neither as unsolicited or as responce to Router Solicitation"

Reply:
Thanks for the thought here.  I also at first thought updating the deprecated leaf description would be useful.  This was debated lightly here:  https://github.com/openconfig/public/pull/900#discussion_r1254769606

However, thinking through this a bit more, since there are differences in implementations due to the ambiguity here I think it's good enough to leave the description as is.  This is because implementations aren't expected to change their behavior for a deprecated leaf and rather should implement the new leaf and enum in this PR.